spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -144,4 +164,82 @@
       end
     end
   end
+
+  # TODO move to separate spec, something like spec/models/header_lookup_spec.rb
+  let(:mesh_term) { "Vocabulary, Controlled" }
+  let(:mesh_query_url) do
+    "https://id.nlm.nih.gov/mesh/sparql?format=JSON&limit=10&inference=true&query=PREFIX%20rdfs%3A%20%3Chttp%3A%2F%2Fwww"\
+    ".w3.org%2F2000%2F01%2Frdf-schema%23%3E%0D%0APREFIX%20meshv%3A%20%3Chttp%3A%2F%2Fid.nlm.nih.gov%2Fmesh%2Fvocab%23%3E"\
+    "%0D%0APREFIX%20mesh2018%3A%20%3Chttp%3A%2F%2Fid.nlm.nih.gov%2Fmesh%3E%0D%0A%0D%0ASELECT%20%3Fd%20%3FdName%0D%0AFROM"\
+    "%20%3Chttp%3A%2F%2Fid.nlm.nih.gov%2Fmesh%3E%0D%0AWHERE%20%7B%0D%0A%20%20%3Fd%20a%20meshv%3ADescriptor%20.%0D%0A%20%"\
+    "20%3Fd%20rdfs%3Alabel%20%3FdName%0D%0A%20%20FILTER(REGEX(%3FdName%2C%27Vocabulary, Controlled%27%2C%20%27i%27))%20%0"\
+    "D%0A%7D%20%0D%0AORDER%20BY%20%3Fd%20%0D%0A"
+  end
+  let(:mesh_api_response) do
```

Comment:
Not requesting the change in this PR, but to make you aware of this just in case: you could probably use vcr to record a cassette here.

Reply:
I've had some problems in the past with VCR so I decided to just go with this. It's not a great reason to avoid VCR though so I should probably use it for the future.